examples/grid.py

```diff
@@ -0,0 +1,62 @@
+import pygame
```

Comment:
In other example files, they import `pygame` as `pg`, I don't like it, but it's better to stay consistent.